CHANGELOG.md

```diff
@@ -8,6 +8,16 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+### Important Changes
+
+- **ref: Deprecate `enableTracing` (12897)**
+
+The `enableTracing` option has been deprecated and will be removed in the next major version. We recommend removing it
+in favor of the `tracesSampleRate` and `tracesSampler` options. If you want to enable performance monitoring, please set
+the `tracesSampleRate` to a sample rate of your choice, or provide a sampling function as `tracesSampler` option
+instead. If you wan't to disable performance monitoring, remove the `tracesSampler` option and set the
+`tracesSampleRate` to `0` or `undefined`.
```

Comment:
```suggestion
instead. If you wan't to disable performance monitoring, remove the `tracesSampler` option and set the
`tracesSampleRate` `undefined`.
```

At least for for node and others, `tracesSampleRate: 0` should still emit transactions if a `sentry-trace` header with a decision to sample (`-1`) arrives. `tracesSampleRate: undefined` should instead not show this behaviour.

Reply:
right